wurst/extensions/UnitExtensions.wurst

```diff
@@ -134,19 +134,36 @@ public function unit.setCreepGuard(bool value)
 
 public function unit.addOrStackItem(item itm) returns bool
     //Stackables
-    if isItemStackable(itm.getTypeId())
-        if not this.isItemFullyStacked(itm.getTypeId()) and this.hasItemById(itm.getTypeId())
-            this.stackItem(itm)
-            return true
-        else if not this.isInventoryFull()
-            this.addItemHandle(itm)
-            return true
+    var result = false

```

Comment:
Is the return value `false` desired value when item is stackable but is removed after the process?